Dependencies/xr/Source/OpenXR/XR.cpp

```diff
@@ -991,6 +1065,60 @@ namespace xr
                         inputSource.AimSpace.Pose.Orientation.W = location.pose.orientation.w;
                     }
                 }
+                
+                // Get joint data
+                if (sessionImpl.HandData.HandsInitialized)
+                {
+                    XrHandJointsLocateInfoEXT jointLocateInfo{XR_TYPE_HAND_JOINTS_LOCATE_INFO_EXT};
+                    jointLocateInfo.baseSpace = m_impl->sessionImpl.SceneSpace;
+                    jointLocateInfo.time = m_impl->displayTime;
+
+                    auto handInfo = sessionImpl.HandData.handsInfo[idx];
+                    XrCheck(sessionImpl.HmdImpl.Extensions->xrLocateHandJointsEXT(handInfo.handTracker, &jointLocateInfo, &handInfo.handJointData.locations));
+                    
+                    auto& inputSource = InputSources[idx];
+                    inputSource.JointsTrackedThisFrame = handInfo.handJointData.locations.isActive;
+                        
+                    // Set up the handJoints vector, skipping the palm joint as webXR doesn't support it
+                    // xrLocateHandJointsEXT will always return the full joint set (or an error), so jointCount should never change
+                    if (inputSource.HandJoints.size() != handInfo.handJointData.locations.jointCount - 1)
```

Comment:
>handInfo.handJointData.locations.jointCount - 1 [](start = 57, length = 47)

nit: given this value is reused multiple locations, it may be worth caching in a variable for legibility